### ISSUE TITLE
Loss does not decrease

### ISSUE THREAD

**Issue by TuZehai** [gantheory/Conv-TasNet#3]
Hi, I am training the model in WSJ0-mix, the the loss does not really decrease as expected. I checked the hyperparameters and the network, and everything looks good to me. Is it possible that there is some bug?

Thanks
![Screenshot from 2019-07-17 16-18-30](https://user-images.githubusercontent.com/29578522/61388054-2ed58880-a8af-11e9-9714-39fcc7c1558a.png)

**Comment by ujscjj** [comment]
hello, you can refer to the codes written by the author of ConTasnet. The code is okay. By the way, I donnot have the wsj0 dataset. Do you konw some links where can download it free?

**Comment by TuZehai** [comment]
> hello, you can refer to the codes written by the author of ConTasnet. The code is okay. By the way, I donnot have the wsj0 dataset. Do you konw some links where can download it free?

I don't think it is possible to download the wsj0 free without a license
